package.json

```diff
@@ -22,65 +22,66 @@
     "linked data",
     "compression"
   ],
-  "main": "dist/main.js",
-  "module": "main.js",
+  "type": "module",
```

Comment:
@Marcus-Mattr one of the primary purposes of this fork was to work around a series of issues while using ESM modules 😄, therefore, we repackaged the module and overwritten the entry point. See this upstream PR for more details: https://github.com/digitalbazaar/cborld/pull/49

However, things may have been changed, please feel free to give it another try.

Reply:
yeah interesting change. Looks like they remove the use of [esm](https://github.com/standard-things/esm#readme) and rely on the native ES module loader instead. wonder if this will fix the slowness issue we encounter before @zhenwenc .

Reply:
Very great point 👍 I forgot the problem was mainly caused by the `esm` package earlier. Since they have removed this usage, the Jest problem in the `platform` repo could have been resolved. 🎉 

Do remember to double-check the build steps (especially in `wallet-sdk`), I think Jest and Webpack loader will not transpile libraries in `node_modules`, you may need to explicitly include the cborld library.

Reply:
just pull it down and play around a bit. feel like you would need some extra config to make it work for pure ESM module and this will be required or all upstream comsumers? i could be wrong though.
Given that we will need to fork regardless because of a couple of extra fix, would make more sense to keep the current approach and the CJS module?

Reply:
Have updated the PR description trying to wrap up everything that was discussed here and in slack 🙌